packages/config/config/devices/0x0098/ct101.json

```diff
@@ -91,9 +102,72 @@
 				}
 			]
 		},
+		"4": {
+			"label": "Power Status",
+			"description": "C-Wire / Battery Status",
+			"valueSize": 1,
```

Comment:
This isn't valid. The min and max cannot both be zero. if this is a limited set of options, please list out the options.

Reply:
Updated.